FWCore/Framework/src/EventProcessor.cc

```diff
@@ -486,13 +483,13 @@ namespace edm {
     ScheduleItems items;
 
     //initialize the services
-    std::shared_ptr<std::vector<ParameterSet> > pServiceSets = processDesc->getServicesPSets();
-    ServiceToken token = items.initServices(*pServiceSets, *parameterSet, iToken, iLegacy, true);
+    auto pServiceSets = processDesc->getServicesPSets();
```

Comment:
Can change `pServiceSets` to `serviceSets`.  There's no copy though...it's a move-construction of `pServiceSets`, which is slightly less efficient then `auto const&`.